AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -9,36 +9,34 @@ import UIKit
 #if canImport(AdyenEncryption)
     import AdyenEncryption
 #endif
+#if canImport(AdyenCardScanner)
+    import AdyenCardScanner
+
+    private let isCardScannerAvailable = true
```

Comment:
Is there a scenario where the AdyenCardScanner is bundled but e.g. the device does not support card scanning? (Thinking about Mac Catalyst, Some iPod touch models maybe?)
Maybe also good to do the iOS version check here (computed property) as well.

Reply:
There is another combined check [here](https://github.com/Adyen/adyen-ios/pull/2045/files#diff-e2b149ce65acfbc811da99a674f2533ac7679d427f89c4864cddde18065c5bf6R39).
Would it help renaming this to `isCardScannerLibraryAvailable`?

Reply:
Makes it more clear what is actually meant with this flag, yes 👌